044-DevOpswithGithubActions/Coach/Challenge01.md

```diff
@@ -0,0 +1,19 @@
+# Challenge 01 : Coach's Guide
```

Comment:
Please add challenge name to each coach guide page as per the challenge-solution-template in the WTH author's guide:
https://github.com/microsoft/WhatTheHack/blob/master/000-HowToHack/WTH-Challenge-Solution-Template.md

Example: 
"Challenge 01: Assessment and Getting Ready for Prerequisites - Coach's Guide"

Also, please ensure Challenge title is consistent in both the Coach Guide ToC and here in the Coach guide for this challenge.